eq.yml

```diff
@@ -30,18 +30,6 @@ resources:
     uri: https://github.com/ONSdigital/eq-ecs-deploy.git
     branch: master
 
-- name: eq-survey-launcher-deploy
```

Comment:
@jonnyshaw89 Does the launcher get deployed within the ecs-deploy module ?

Reply:
or is this just because it is not needed for production ?

Reply:
Do we use this resource?